Glideslope.json

```diff
@@ -43578,6 +43578,19 @@
    "glideslope": 3.00340776,
    "receptionRange": 10
  },
+ {
+   "name": "GS",
+   "airportICAO": "ZPLC",
+   "identifier": "ILG",
+   "associatedRunwayNumber": "16",
+   "frequency": 108500,
+   "elevation": 6204,
+   "latitude": 23.74877778,   
+   "longitude": 100.01944371,
+   "bearing": 154.57,
+   "glideslope": 3.0001314,
```

Comment:
Hi! The glideslope should be 3.2° here, so need to change to 3.2015457
![image](https://user-images.githubusercontent.com/37546244/87382451-8e04d080-c5c9-11ea-8333-e737b58b0d30.png)

Reply:
@AnthonyCK that looks to me a 3.2 degree runway downgrade 🤷

Reply:
Yes, this is a 3.2deg glideslope. But the separator is not taken into the GS.json the format for glideslope is three didgits GS and 5 digits bearing without any separation.
So 3.21 GS and 123.45 bearing becomes 32112345 in Gs.jason

Reply:
ok, then it seems the editor has made an error with the PAPI angle of the airport.

Reply:
Fixed in airports